RELEASING

```diff
@@ -0,0 +1,25 @@
+# Scala.js Release Process
```

Comment:
If you're using Mardown syntax, you should use a `.md` extension. Especially for that numbered list with 1's everywhere down below ;)